README.rst

```diff
@@ -1,36 +1,83 @@
+* ``depends_on_register`` - Determines whether to add comments about a dependency on [Register](https://github.com/adafruit/Adafruit_CircuitPython_Register). Leave empty if the library won't use Register.
+* ``author`` - Who you are! Sets the copyright to you.
+* ``github_user`` - GitHub user or organization which will host this repo. For example, Adafruit funded libraries should say "adafruit" here.
+* ``library_prefix`` - Used to prefix the code to the organization creating the library. For example, Adafruit supported libraries should say "adafruit" here. Do not add a - or _.
+* ``company`` - Used to give Copyright credit to the company funding the library. For example, Adafruit funded libraries should say "Adafruit Industries" here.
+
+Post Generation Cleanup
+------------------------
+
+After generation, make sure to glance over the files to make sure they
+autogenerated as you expect (such as capitalization). There are a few places
+with ``.. todo::`` that should also be taken care of. After adding or updating
+the information requested, make sure the ``.. todo::`` text is removed. Like this:
+
+.. code::
+
+    # Before Cleanup
+    .. todo:: Describe what the module does
+
+.. code::
+
+    # After Cleanup
+    This library talks to the AM4Z-1NG sensor. Typical use is for robot friends.
+
+Windows Users
+==============
+
+Due to the development nature of cookiecutter, there are some limitations when using with Windows.
+
+Cookiecutter Installation
+--------------------------
+
+The Python enviornment can be tricky sometimes in Windows. Use this documentation page for steps and tips on Windows installation: `Cookiecutter Installation - Windows <https://cookiecutter.readthedocs.io/en/latest/installation.html#windows>`_
+
+
+<library>.py & /examples/<library>_simpletest.py File Generation
+------------------------------------------------------------------
+
+Cookiecutter was developed for use in \*\nix/OSX enviornments. When implementing prompt based configuration for things like filenames, special characters were used for programatic detection and formatting. 
+
+.. code-block::
+
+    {% if cookiecutter.library_prefix %}{{ cookiecutter.library_prefix | lower }}_{% endif %}{{ cookiecutter.library_name | lower }}.py 
+
+As such, Windows will block the use of these special characters in filenames. So when cookiecutter pulls the Adafruit CircuitPython template, the <library>.py and /examples/<library>_simpletest.py files are not created. This adds an extra step. Simply copy and existing librariy's .py files (and structure if  making a "package"), and change the propmted values (e.g. author name, library name, documentation information, etc).
+
+.. note::
+    The above is from experience with using cookiecutter within a Windows native setup. This may not be applicable when using Windows Subsystem for Linux (WSL) or any *nix-For-Windows utilities.

```

Comment:
Does `any *nix-For-Windows` need the `\*\` with it like you did at the beginning of this section?

Reply:
I don't _think_ it does. The escaping at the beginning is because `*nix/OSX` makes the `*` become a hyperlink (to itself). This doesn't appear to be happening on `*nix-For-Windows` At least it doesn't for me.